schema.sql

```diff
@@ -90,3 +90,10 @@ CREATE TABLE curation_rewards (
   comment_permlink VARCHAR(255),
   CONSTRAINT uc_curation_reward UNIQUE (curator, comment_author, comment_permlink)
 );
+
+CREATE TABLE reblogs (
+  created_at TIMESTAMP,
+  account VARCHAR(16) NOT NULL,
+  author VARCHAR(16),
+  permlink VARCHAR(255)
```

Comment:
We should have unique constraint for `account, author, permlink`.

Reply:
Hum, here I've not added the constraint here because i'm not 100% sure if its possible to reblog multiple time the same post.